.env

```diff
@@ -56,6 +56,10 @@ _APP_SMTP_PORT=1025
 _APP_SMTP_SECURE=
 _APP_SMTP_USERNAME=
 _APP_SMTP_PASSWORD=
+_APP_PHONE_PROVIDER=mock
```

Comment:
The DSN approach that @christyjacob4 is implementing for databases might work better here and possibly for storage as well.